src/frontend/index.scss

```diff
@@ -25,6 +25,13 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   user-select: none;
+  /* Frameless window defaults */
+  --titlebar-color: var(--body-background);
+  --titlebar-symbol-color: var(--accent);
+  --overlay-controls-height: env(titlebar-area-height, 39px);
```

Comment:
I wonder how this plays with system settings, like if a windows users changes their system to have bigger buttons I imagine this will not follow what the user wants, maybe we need to make this configurable in the Accessibility screen?

Maybe not for this PR but for a future improvement

Reply:
Good question, I'll look into it.

Reply:
Okay, so the custom WindowControls component actually plays along nicely and scales with the zoom factor set in the accessibility settings. For the native overlay controls, I could add some code to adjust the height according to the zoom factor in the handler for the `setZoomLevel` event (main.ts#250-254), but it would require having access to the existing window props because `BrowserWindow` does not have a method to read them out.

What are your thoughts on this?

Reply:
Okay, I gave this a try but it looks like the size of the native controls cannot be adjusted at runtime like the documentation suggests (at least not on Windows). However, it also occurred to me that if someone is using the zoom function in the app they are likely also already using the zoom function built into Windows that allow making everything, including window controls, bigger.

If this is a serious concern, we could make the native window controls optional. Basically, when frameless mode is enabled, we could show another checkbox for using native window controls if this is supported by the OS.

Reply:
We can see how it is now and wait for feedback, don't want that to be a blocker since I don't even know what the right thing to do there haha.

Reply:
Well, adding another toggleswitch for this really isn't that complicated, already did so last night. Can push if you want me to.